SPECS/llvm/llvm.spec

```diff
@@ -49,7 +49,7 @@ cmake -G Ninja                              \
       -DLLVM_LINK_LLVM_DYLIB=ON             \
       -DLLVM_INCLUDE_TESTS=ON               \
       -DLLVM_BUILD_TESTS=ON                 \
-      -DLLVM_TARGETS_TO_BUILD="host;AMDGPU;BPF" \
+      -DLLVM_TARGETS_TO_BUILD="host"        \
```

Comment:
IMO, dropping `BPF` here would cause a problem for almost all the `eBPF` related tools, because if we don't have a BPF traget registered, llvm wont be able to process it.